recipes/h5sparse-tensor/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set version = "0.2.2" %}
+
+package:
+  name: "h5sparse-tensor"
+  version: {{ version }}
+
+source:
+  url: https://github.com/tvandera/h5sparse/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b73a08695d40266f70b9cdfbceb6d61546e32f808d7817e854b764a7890ee76d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - h5py
+    - numpy
+    - python >=3.6
+    - scipy
+    - six
+    - pip
+    - nose
```

This should be sufficient for host
```suggestion
  host:
    - python >=3.6
    - pip
```